Linter gives 6 warnings

**Is your feature request related to a problem? Please describe.**
In going through the README, it raises 6 errors in running `npm run lint`

**Describe the solution you'd like**
Wanted: linting returns with "clean". That is, no warnings or errors.

How: suppress the warnings. _Don't_ change the contracts here, since they've long been deployed. 

**Linter log**

```console
trentmc@trentmcLnv:~/code/contracts$ npm run lint

> @oceanprotocol/contracts@0.5.10 lint
> npm run lint:eslint && npm run lint:ethlint


> @oceanprotocol/contracts@0.5.10 lint:eslint
> eslint --ignore-pattern '/coverage/' .


> @oceanprotocol/contracts@0.5.10 lint:ethlint
> solium -d ./contracts/


contracts/balancer/BNum.sol
  116:8    warning    Assignment operator must have exactly single space on both sides of it.    operator-whitespace
  134:8    warning    Assignment operator must have exactly single space on both sides of it.    operator-whitespace
  135:8    warning    Assignment operator must have exactly single space on both sides of it.    operator-whitespace
  137:8    warning    Assignment operator must have exactly single space on both sides of it.    operator-whitespace

contracts/templates/DataTokenTemplate.sol
  249:12    warning    Avoid using 'block.timestamp'.    security/no-block-members
  285:12    warning    Avoid using 'block.timestamp'.    security/no-block-members

✖ 6 warnings found.
```

--- a/contracts/balancer/BNum.sol
+++ b/contracts/balancer/BNum.sol
@@ -113,7 +113,7 @@ contract BNum is BConst {
         require(base >= MIN_BPOW_BASE, 'ERR_BPOW_BASE_TOO_LOW');
         require(base <= MAX_BPOW_BASE, 'ERR_BPOW_BASE_TOO_HIGH');
 
-        uint whole  = bfloor(exp);   
+        uint whole = bfloor(exp);
         uint remain = bsub(exp, whole);
 
         uint wholePow = bpowi(base, btoi(whole));
@@ -131,10 +131,10 @@ contract BNum is BConst {
         returns (uint)
     {
         // term 0:
-        uint a     = exp;
-        (uint x, bool xneg)  = bsubSign(base, BONE);
+        uint a = exp;
+        (uint x, bool xneg) = bsubSign(base, BONE);
         uint term = BONE;
-        uint sum   = term;
+        uint sum = term;
         bool negative = false;
 
 

--- a/contracts/templates/DataTokenTemplate.sol
+++ b/contracts/templates/DataTokenTemplate.sol
@@ -240,12 +240,12 @@ contract DataTokenTemplate is IERC20Template, ERC20 {
         }
         uint256 totalFee = communityFee.add(marketFee);
         transfer(_minter, amount.sub(totalFee));
-
         emit OrderStarted(
             consumer,
             msg.sender,
             amount,
             serviceId,
+            /* solium-disable-next-line */
             block.timestamp,
             mrktFeeCollector,
             marketFee
@@ -275,13 +275,14 @@ contract DataTokenTemplate is IERC20Template, ERC20 {
                 transfer(consumer, amount),
                 'DataTokenTemplate: failed to finish order'
             );
-
+        
         emit OrderFinished(
             orderTxId, 
             consumer, 
             amount, 
             serviceId, 
             msg.sender,
+            /* solium-disable-next-line */
             block.timestamp
         );
     }

